main.go

```diff
@@ -15,34 +15,38 @@ package main
 
 import (
 	"memphis-control-plane/analytics"
+	"memphis-control-plane/background_tasks"
```

Comment:
move this import to be after the DB import

Reply:
When I save the file it moves up automaticly